lib/solidus_paypal_commerce_platform.rb

```diff
@@ -4,6 +4,6 @@
 require 'solidus_support'
 
 require 'solidus_paypal_commerce_platform/version'
-require 'solidus_paypal_commerce_platform/environment'
+require 'solidus_paypal_commerce_platform/config'
```

Comment:
Can you attach all the changes related to renaming to this commit?
That would leave the next one with just the new configuration methods without the "noise" of the renaming.

Reply:
I think I get what you mean here, but take a look at the commits just to make sure! Thanks!